src/utils/user.ts

```diff
@@ -0,0 +1,48 @@
+import cheerio, { CheerioAPI } from "cheerio";
+
+import axios from "axios";
+
+interface UserProfile extends CheerioAPI {
+  kind: "UserProfile";
+}
+
+export interface User {
+ name: string;
+ pseuds: string;
+ url: string;
+ id: string;
+ joined: string;
+ bio: string;
```

Comment:
This might be undefined, if I understand correctly, so you might want to type it as `bio?: string`.

You might also want to call it `bioHtml` since if not people might expect just the text (unless you do want to return just the text).

Reply:
Might put "add ability to do text or html" on the back burner for later since people might want one or the other?

Would `bioHtml: string | null` be what you need it to be (and/or do I also need/want the `?`), I forgot I had it as `string | null` the first time